src/routes/userRoutes/authRoutes.js

```diff
@@ -0,0 +1,12 @@
+const { default: fastify } = require('fastify')
```

Comment:
Let's move authRoutes.js into it's own folder, with an index.js to condense all the filed that will be required. This will help with scalability if in the future we decide to add any more authentication strategies.